presubmit.py

```diff
@@ -338,24 +339,21 @@ def main() -> int:
     os.chdir(_SRC_ROOT)
     changed_files = get_changed_files()
 
-    if args.command == 'format':
-        success = yapf(changed_files, False)
-        return bool_to_returncode(success)
-
-    if args.command == 'lint':
-        success = lint(changed_files)
-        return bool_to_returncode(success)
-
-    if args.command == 'typecheck':
-        success = pytype(changed_files)
-        return bool_to_returncode(success)
-
-    if args.command == 'licensecheck':
-        success = license_check(changed_files)
-        return bool_to_returncode(success)
-
-    if args.command == 'test_changed_integrations':
-        success = test_changed_integrations(changed_files)
+    command_check_mapping = {
```

Comment:
Does this code work if we just run
make presubmit

Shouldn't this block be in "if args.command"

Reply:
It does work, but I've changed it to avoid unnecessary looping.